lang/cs/Org.Apache.REEF.Client/YARN/YARNREEFClient.cs

```diff
@@ -83,11 +90,26 @@ public IDriverHttpEndpoint SubmitAndGetDriverUrl(IJobSubmission jobSubmission)
             Launch(jobSubmission, driverFolderPath);
 
             var pointerFileName = Path.Combine(driverFolderPath, _fileNames.DriverHttpEndpoint);
+            var jobSubmitionResultImpl = new JobSubmissionResultImpl(this, pointerFileName, true);
 
-            var httpClient = new HttpClientHelper();
-            _driverUrl = httpClient.GetDriverUrlForYarn(pointerFileName);
+            Logger.Log(Level.Info, string.Format(CultureInfo.CurrentCulture,
```

Comment:
Same as above: Please pass the format string to the logger instead of forcing its expansion here.